.github/workflows/maven.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
```

Comment:
Wouldn't for consistencies sake it be better to name the file build.yml? That keeps it consistent with our other Actions and we know what to look for across repositories.

Reply:
Renamed file